src/site/content/en/blog/user-preference-media-features-headers/index.md

```diff
@@ -0,0 +1,184 @@
+
+Additionally, and specifically for `prefers-color-scheme`, sites by all means want to avoid a
+[flash of inaccurate color theme](https://css-tricks.com/flash-of-inaccurate-color-theme-fart/).
+
+The `Sec-CH-Prefers-Color-Scheme` client hint header is the first of a
+[series of user preference media features client hints headers](https://wicg.github.io/user-preference-media-features-headers/)
+that aims to solve this issue.
+
+### Background on client hints
+
+[HTTP Client Hints](https://datatracker.ietf.org/doc/html/rfc8942) defines an `Accept-CH` response
+header that servers can use to advertise their use of request headers for proactive content
+negotiation, colloquially referred to as client hints. The
+[User Preference Media Features Client Hints Headers](https://wicg.github.io/user-preference-media-features-headers/)
+proposal defines a set of client hints aimed at conveying user preference media features. These
+client hints are named after the corresponding user preference media feature that they report on.
+For example, the currently preferred color scheme as per `prefers-color-scheme` is reported via the
+aptly named `Sec-CH-Prefers-Color-Scheme` client hint.
+
+### Background on critical client hints
+
+The client hints proposed in
+[User Preference Media Features Client Hints Headers](https://wicg.github.io/user-preference-media-features-headers/)
+will presumably most commonly be used as
+[Critical Client Hints](https://tools.ietf.org/html/draft-davidben-http-client-hint-reliability-02).
+Critical Client Hints are Client Hints which meaningfully change the resulting resource. Such a
+resource should be fetched consistently across page loads (including the _initial_ page load) to
+avoid jarring user-visible switches.
+
+### Client hint syntax
+
+User preference media features consist of a name (like `prefers-reduced-motion`) and allowed values
+(like `no-preference` or `reduce`. Each client hint header field is represented as
+[Structured Headers for HTTP](https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-15)
+object containing an
+[item](https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-15#section-3.3) whose value
+is a [string](https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-15#section-3.3.3). For
+example, to convey that the user prefers a dark theme and reduced motion, the client hints look like
+in the example below.
+
+```bash
+Sec-CH-Prefers-Color-Scheme: "dark"
+Sec-CH-Prefers-Reduced-Motion: "reduce"
+```
+
+The CSS equivalent of the information conveyed in the above client hints is
+`@media (prefers-color-scheme: dark) {}` and `@media (prefers-reduced-motion: reduce) {}`
+respectively.
+
+## Complete list of the client hints
+
+{% Aside %} While the
+[User Preference Media Features Client Hints Headers](https://wicg.github.io/user-preference-media-features-headers/)
+proposal defines a set of client hints, Chromium for now only supports
```

Comment:
Nit: "for now" should be replaced by "at the time of writing".

Reply:
True. Done.